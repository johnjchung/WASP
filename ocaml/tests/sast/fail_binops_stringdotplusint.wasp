Endpoint stringDotPlusInt(GET:) {
    string Test = "Hello World!!!" .+ 5;
}
