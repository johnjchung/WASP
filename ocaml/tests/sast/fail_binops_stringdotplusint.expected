Fatal error: exception Type_checking.BinopTypeError("THE BINARY OPERATION OF TYPES IS NOT ALLOWED: string .+ int")
